### ISSUE TITLE
Download link on SomeRandomDude.com is outdated

### ISSUE THREAD

**Issue by lorenzos** [somerandomdude/Iconic#28]
I was opening an issue about CSS class namespacing when I found an already closed one. So I realized I had just downloaded a very outdated version of your icons from your personal website. The download link of that outdated package is:

> https://github.com/downloads/somerandomdude/Iconic/iconic.zip

Consider updating that package or, **better**, you can link directly to the one served by GitHub, which will be **always up-to-date**:

> https://github.com/somerandomdude/Iconic/archive/master.zip

**Comment by somerandomdude** [comment]
Good call. I'll need to update the download. The reason why they're different is that I don't include all the different variations of PNG files in the Github repo. So I need to have separate downloads. Thanks for catching this!

**Comment by chid** [comment]
Right now there doesn't seem to be any download link / reference to open iconic on the https://useiconic.com/ website is that the way it was meant to be?

**Comment by somerandomdude** [comment]
The link for the new Open Iconic will be live very soon. Lots will be changing around this project very soon.

**Comment by chid** [comment]
Exciting :+1:
